CODEOWNERS

```diff
@@ -114,10 +114,17 @@ ttnn/ @eyonland @arakhmati @cfjchu @xanderchin @TT-BrianLiu @ayerofieiev-tt @dma
 ttnn/**/kernels/ # Removes the owners above from owning kernels unless specified afterwards
 ttnn/setup.py @tt-rkim
 ttnn/CMakeLists.txt @tt-rkim @ayerofieiev-tt
+ttnn/cpp/ttnn/operations/ccl/ @SeanNijjar @TT-BrianLiu
+ttnn/cpp/ttnn/operations/pool/ @mywoodstock @pavlejosipovic
+ttnn/cpp/ttnn/operations/conv2d/ @mywoodstock @pavlejosipovic
```

Comment:
@sankarmanoj-tt will also need to be added here. He is also part of the conv team.

Reply:
Make the change and propose, this is already merged